G/GTPSA/build_tarballs.jl

```diff
@@ -0,0 +1,65 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "GTPSA"
+version = v"1.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/mattsignorelli/gtpsa.git", "ed2c14eb096a9ecac3f483f9914c0c679765496d")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd gtpsa/
+if [[ $target == *"-apple-"* ]]
+then
+CC=gcc
+CXX=g++
+ln /workspace/destdir/lib/liblapack32.dylib /workspace/destdir/lib/liblapack.dylib
+elif [[ $target == *"linux"* ]]
+then
+ln /workspace/destdir/lib/liblapack32.so /workspace/destdir/lib/liblapack.so
```

Comment:
While I guess this is fine, I believe an alternate way would be to add
```
set( BLA_SIZEOF_INTEGER 4 )
```
to the upstream CMake before your `find_package( LAPACK )` call, and then ensure that the most recent CMake is installed in the build root here. You can do that by adding
```
# we need a more recent cmake version for suitesparse and use the jll instead
apk del cmake
```
to the start of the build script, and then adding a new dependency for
```
HostBuildDependency(PackageSpec(; name="CMake_jll", version = v"3.24.3"))
```
(or another CMake version greater than 3.22).

Reply:
I tried this, and ensured the CMake version was > 3.22, and it was giving an error in the linking stage of make because it could not find -llapack, even though CMake was able to find it. So I believe the link with the expected liblapack name is necessary